Anishchenko_V/HW6_collections/task2_check_login.py

```diff
@@ -0,0 +1,6 @@
+login = input('Enter login: ')
```

Comment:
good job
look at this way

user_name = input('Hello, please input your Log in:')

while user_name != 'First':
    user_name = input('Error: wrong username, please try one more time. Username:')
else:
    print('Greeting. Access granted!!!', user_name)